src/models/Cart.js

```diff
@@ -3,75 +3,32 @@ const mongoose = require('mongoose');
 const SingleCartItemSchema = new mongoose.Schema(
   {
     title: { type: String, required: true },
-    image: { type: String, required: true },
+    author: { type: String, required: true },
+    coverImageUrl: { type: String, required: true },
     price: { type: Number, required: true },
-    book: {
-      type: mongoose.Schema.ObjectId,
-      ref: 'Book',
-      required: true,
-    },
-    isAvailable: { type: Boolean, required: true, default: true },
+    book: { type: mongoose.Schema.ObjectId, ref: 'Book', required: true },
+    isAvailable: { type: Boolean, default: true },
   },
   { timestamps: true }
 );
 
 const CartSchema = new mongoose.Schema(
   {
-    tax: {
-      type: Number,
-      required: true,
-      min: [0, 'Tax cannot be negative'],
-    },
-    shippingFee: {
-      type: Number,
-      required: true,
-      min: [0, 'Shipping fee cannot be negative'],
-    },
-    total: {
-      type: Number,
-      required: true,
-      min: [0, 'Total cannot be negative'],
-    },
+    tax: { type: Number, default: 0, min: 0 },
+    shippingFee: { type: Number, default: 0, min: 0 },
+    total: { type: Number, default: 0, min: 0 },
     orderItems: [SingleCartItemSchema],
     status: {
       type: String,
-      enum: ['pending', 'failed', 'paid', 'delivered', 'canceled'],
+      enum: ['pending', 'paid', 'failed'],
       default: 'pending',
     },
-    createdBy: {
-      type: mongoose.Types.ObjectId,
-      ref: 'User',
-      required: true,
-    },
-    clientSecret: {
-      type: String,
-      required: true,
-    },
-    paymentIntentId: {
-      type: String,
-    },
-    expiresAt: {
-      type: Date,
-      default: () => Date.now() + 24 * 60 * 60 * 1000,
-      index: { expireAfterSeconds: 0 },
-    },
+    createdBy: { type: mongoose.Schema.ObjectId, ref: 'User', required: true },
+    clientSecret: { type: String, default: '' },
+    paymentIntentId: { type: String },
+    expiresAt: { type: Date, default: () => Date.now() + 24 * 60 * 60 * 1000 },
```

Comment:
Can you test this to see if it runs once at server start?

I think we want a new date: 
expiresAt: { type: Date, default: () => new Date(Date.now() + 24 * 60 * 60 * 1000) },